lib/js/src/manager/file/filetypes/SdlFile.js

```diff
@@ -158,6 +159,24 @@ class SdlFile {
     isStaticIcon () {
         return this._isStaticIcon;
     }
+
+    /**
+     * Sets the overwrite property for an SdlFile
+     * @param {Boolean} fileName - A boolean value that indicates if the file can be overwritten
```

Comment:
```suggestion
     * @param {Boolean} overwrite - A boolean value that indicates if the file can be overwritten
```
Fixes issues found by linter.